### PR TITLE
✨ Add variable Button component

### PR DESCRIPTION
**Pull request by Naasgul**
We added the button component with prop.types and integration on Storybook. The Styling is still in progress. 

There seems to be a problem with the node.js checks, maybe someone has a clue what causes the failed test. 

**UPDATE:**
It seems that this line of the test causes the error:
``
Treating warnings as errors because process.env.CI = true.
Most CI servers set it automatically.

Failed to compile.
`` (found in the log of the node.js-test) 
I'll try to set process.env.CI =false to make the tests work again

**UPDATE:**

This article helped me to solve the node-test issue: https://docs.netlify.com/configure-builds/troubleshooting-tips/#build-fails-on-warning-message

### CODE REVIEW

**Comment by boldCM** [comment]
lgtm 🎉
your're node build on github actions is not working...but everything else is 👍🏻

**Review by boldCM — APPROVED** [review]
lgtm 🎉
your're node build on github actions is not working...but everything else is 👍🏻

**Comment by Naasgul** [comment]
> lgtm 🎉
> your're node build on github actions is not working...but everything else is 👍🏻



> lgtm 🎉
> your're node build on github actions is not working...but everything else is 👍🏻

Thanks for the review - yes, there seems to be a problem with running the "npm run build" script which works fine in VSCode tho

File: client/src/components/Button.js [hunk 1]
```diff
@@ -1,33 +1,43 @@
...
+    height: "1rem",
+    font: "Oxygen-Regular",
+  },
+  medium: {},
+  large: {
+    width: "40vw",
+    height: "2rem",
+    font: "Oxygen-Bold",
+  },
+};
```
Comment (leonp5) [inline client/src/components/Button.js:17]: Nice way to make sizes 'globally' accessible :)

**Review by leonp5 — APPROVED** [review]

**Comment by bwnstck** [comment]
Build Script is missing in root package.json that's why the test fails

You could try something like 

```
build: "cd client &&  npm run build"
```

Done on mobile :)

**Comment by Naasgul** [comment]
> Build Script is missing in root package.json that's why the test fails
> 
> You could try something like
> 
> ```
> build: "cd client &&  npm run build"
> ```
> 
> Done on mobile :)


> Build Script is missing in root package.json that's why the test fails
> 
> You could try something like
> 
> ```
> build: "cd client &&  npm run build"
> ```
> 
> Done on mobile :)

Thank you for the tip, but we got the "build-script" in the package.json of the root directory:
```
    "build": "cd client && npm run build && npm run build-storybook",
```

**Comment by bwnstck** [comment]
Looks like you fixed it :)
I was on the wrong branch i guess

... The fix you implemented is actually a bit hacky... Looks like you have some unfixed warnings... Better would be to fix those instead of telling the CI to ignore them ;)

**Pull request merged**